### DISCUSSION TITLE
Bug in applying coupon code

### DISCUSSION THREAD

**Discussion by mohitkumar29** [swellstores/community#548] [Help] [answered]
I am following what's mentioned [here](https://developers.swell.is/frontend-api/coupons#apply-a-coupon) to apply coupon code to the cart, instead it's applying the **GIFT CARD** which has a different method of it's own mentioned [here](https://developers.swell.is/frontend-api/gift-cards#apply-a-gift-card). 

one thing to note is that both the gift card that is being applied instead of the coupon and the coupon code that i am trying to apply are the same (ABC123 vs ABC123), is that what's causing this? If that is indeed the case then this should be fixed as there are separate methods for both.

**Comment by sags95** [comment, marked as answer]
I will file this with the team to investigate what's going wrong

**Reply by virajsoni06** [reply]
@sags95 any update on this?

**Reply by mohitkumar29** [reply]
@sags95 why was this closed? It's still an issue, please re-open this and provide some solution.

**Reply by logeshswell** [reply]
@mohitkumar29 @virajsoni06 Our engineers are yet to work on this issue. I will keep you posted if we get any update on the fix.
